pkg/cgroup/cgroup_pids.go

```diff
@@ -0,0 +1,76 @@
+package cgroup
+
+import (
+	"errors"
+	"math"
+	"os"
+	"strings"
+
+	"golang.org/x/sys/unix"
+)
+
+func statPidsV2(dirPath string, stats *Stats) error {
+	current, err := getCgroupParamUint(dirPath, "pids.current")
+	if err != nil {
```

Comment:
When will `pids.current` not exist in cgroupv2?

Reply:
If pids controller is not enabled. But I'm not sure if that could be the case for k8s.